odl/discr/lp_discr.py

```diff
@@ -293,8 +303,9 @@ def element(self, inp=None, **kwargs):
         elif inp in self.dspace:
             return self.element_type(self, inp)
         elif callable(inp):
+            vectorized = kwargs.pop('vectorized', True)
             # uspace element -> discretize
-            inp_elem = self.uspace.element(inp)
+            inp_elem = self.uspace.element(inp, vectorized=vectorized)
```

Comment:
At some point, we need to rethink the practice of just "handing down" `kwargs`, especially in cases as this one where there are multiple possible places where to put the arguments.
A viable solution to that issue is to use "named" (`args` and) `kwargs`, e.g. `interp_kwargs` in the interpolation (just an example).

Here it's still somewhat okay but on the boundary to bug-prone. And increasingly tricky to explain in the docs. If there are more things like this we should consider being more specific.